entrypoint.sh

```diff
@@ -273,7 +273,7 @@ is_kernel_module_loaded() {
 
 is_granted_linux_capability() {
 
-  if capsh --print | grep -Eq "^Current: = .*,?${1}(,|$)"; then
+  if capsh --print | grep -Eq "^Current: ?=? .*,?${1}(,|$)"; then
```

Comment:
The expression `?=?` is used in order to make space before `=` optional as well.